src/gui/accountstate.cpp

```diff
@@ -441,6 +461,17 @@ void AccountState::fetchNavigationApps(){
     job->getNavigationApps();
 }
 
+void AccountState::trySignIn()
+{
+    _signInTimer.stop();
```

Comment:
are you sure you want to stop the timer here ?
according to the call to `QTimer::singleShot(0, this, &AccountState::trySignIn);` that looks like the timer will never result in a call to `QTimer::timeout`

Reply:
@mgallien Well, because the single-shot is called with `0` interval, that means the timeout will result in a call for sure, but, I agree that this approach is not reliable. I should have added a wrapper method around `trySignIn` and bound the `QTimer::timeout` to it.

Reply:
@mgallien Good catch!